octoprint_mrbeam/__init__.py

```diff
@@ -335,14 +345,29 @@ def _on_laserhead_ready(self, *args, **kwargs):
         self._laserhead_ready = True
         self._try_to_connect_laser()
 
+    def _on_laser_cutter_profile_initialized(self, *args, **kwargs):
```

Comment:
If we don't need the *args, **kwargs, is it possible to remove them? like
```suggestion
    def _on_laser_cutter_profile_initialized(self, _, _):
```
Or maybe just remove it from the docstring

Reply:
This is fine as we can add any relevant arguments to this later. This is similar to other event handlers we are using elsewhere.

Reply:
I will remove from docstring though